charts/flyte-deps/templates/webhook/service.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.webhook.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "flyte-pod-webhook.name" . }}
+  namespace: {{ template "flyte.namespace" . }}
+  {{- with .Values.webhook.service.annotations }}
+  annotations: {{ tpl (toYaml .) $ | nindent 4 }}
+  {{- end }}
+spec:
+  type: ExternalName
```

Comment:
We should use clusterIP for sandbox, and ExternalName for single binary (flyte-sandbox-lite or demo). Probably make it configurable.

Reply:
okay, sounds good.